src/dash-table/dash/DataTable.js

```diff
@@ -858,6 +860,7 @@ export const propTypes = {
     tooltip: PropTypes.objectOf(
         PropTypes.oneOfType([
             PropTypes.exact({
+                use_with: PropTypes.oneOf(['data', 'header']),
```

Comment:
Backward compatibility addition of `use_with` to the `tooltip` prop -- app creator can use it to define whether the default tooltip will be visible on data, headers, or both.

Reply:
Can we add an explicit `'both'` to go along with the implicit one, and just label that as the default? Even though it won't be set as such since it's nested, I think people would in some cases appreciate being able to set this explicitly.

Reply:
Sure https://github.com/plotly/dash-table/pull/831/commits/71e7ce9b7bbc2752fc121afba4e74a2641061724